modules/contact-form/admin.php

```diff
@@ -37,7 +37,7 @@ function grunion_admin_css() {
 	if ( is_null( $current_screen ) ) {
 		return;
 	}
-	if ( ! in_array( $current_screen->id, array( 'edit-feedback', 'jetpack_page_omnisearch', 'dashboard_page_omnisearch' ) ) ) {
+	if ( ! in_array( $current_screen->id, array( 'edit-feedback' ) ) ) {
```

Comment:
Since we're no longer testing against many values let's drop the more expensive array and in_array usages and change this to 
```php
if ( 'edit-feedback' !== $current_screen->id ) {
```